pyftpdlib/handlers.py

```diff
@@ -3464,6 +3477,11 @@ def get_ssl_context(cls):
                 if not cls.keyfile:
                     cls.keyfile = cls.certfile
                 cls.ssl_context.use_privatekey_file(cls.keyfile)
+                if cls.tls_mutual_authentication:
+                    cls.ssl_context.set_verify(VERIFY_PEER | VERIFY_FAIL_IF_NO_PEER_CERT | VERIFY_CLIENT_ONCE, cls.verify_certs_callback)
```

Comment:
`flake8` is also complaining about this line being too long, but I dunno what @giampaolo 's preferred line-wrapping style is ;-)